client/routes.js

```diff
@@ -1,23 +1,69 @@
-angular.module('probedock.routes', [ 'ui.router' ])
+angular.module('probedock.routes', [ 'ui.router', 'ui.router.title' ])
 
   .config(function($stateProvider, $urlRouterProvider) {
+    var titleElementLimit = 25;
+
+    function buildTitle($stateParams) {
+      var title = 'Probe Dock';
+
+      if ($stateParams.orgName) {
+        title += ' > ';
+
+        if ($stateParams.orgName.length > titleElementLimit) {
+          title += $stateParams.orgName.substr(0, titleElementLimit - 3) + '...'
+        }
+        else {
+          title += $stateParams.orgName;
+        }
+      }
+
+      _.each([].slice.call(arguments, 1), function(part) {
```

Comment:
Nevermind. However, it's better to use `Array.prototype.slice` as it doesn't instantiate an array.